lgsf/scrapers/base.py

```diff
@@ -97,3 +100,193 @@ def save_raw(self, filename, content):
     def save_json(self, obj):
         file_name = "{}.json".format(obj.as_file_name())
         self._save_file("json", file_name, obj.as_json())
+
+
+class CodeCommitMixin:
```

Comment:
Had a stab at pulling out the codecommit logic into a mixin for the scrapers. Not sure it's the best way of doing it, and haven't made it clear what methods the child classes need to implement, but thought I would get a better handle on whether it was a working system when I do the polling station scrapers.